packages/store/src/decorators/select/select.ts

```diff
@@ -0,0 +1,34 @@
+import { Observable } from 'rxjs';
+
+import { createSelectObservable, createSelectorFn } from './symbols';
+import { StoreValidators } from '../../utils/store-validators';
+import { NGXS_DECORATOR } from '../symbols';
+
+/**
+ * Decorator for selecting a slice of state from the store.
+ */
+export function Select(rawSelector?: any, ...paths: string[]): PropertyDecorator {
+  return function(target: any, propertyKey: string | symbol): void {
+    StoreValidators.checkExtensibleBeforeDecorate(target, NGXS_DECORATOR.SELECT);
+
+    const name: string = propertyKey.toString();
+    const selectorId = `__${name}__selector`;
+    const selector: any = createSelectorFn(name, rawSelector, paths);
```

Comment:
Does this need to be cast as `any`? Surely the inferred type would be better?
```suggestion
    const selector = createSelectorFn(name, rawSelector, paths);
```

Reply:
done